articles/install-guide/update.md

```diff
@@ -45,25 +45,37 @@ Regardless of whether you are using C# or Python to host Q# operations, follow t
     ```
 
     You can find more details on specifying target frameworks [here](https://docs.microsoft.com/dotnet/standard/frameworks#how-to-specify-target-frameworks).
-5. Save and close all files in your solution
-6. Select **Tools** -> **Command Line** -> **Developer Command Prompt**
-7. For each project in the solution, run the following command:
+
+5. In each of the .csproj files, set the SDK to `Microsoft.Quantum.Sdk`, as indicated in the line below. Please notice that the version number should be the latest available, and you can determine it by reviewing the [release notes](https://docs.microsoft.com/quantum/relnotes/).
+
+    ```xml
+    <Project Sdk="Microsoft.Quantum.Sdk/0.11.2005.2905">
+    ```
+
+6. Save and close all files in your solution.
+7. Select **Tools** -> **Command Line** -> **Developer Command Prompt**.
+8. For each project in the solution, run the following command to **remove** this package:
+
+    ```dotnetcli
+    dotnet remove [project_name].csproj package Microsoft.Quantum.Development.Kit
```

Comment:
Since this is in Visual Studio, would it make sense to use the built-in Package Management Console instead of the external command prompt? Either should work, of course, just wondering which feels most in line with VS usage.

Reply:
I'll add a note to mention the alternative.